ui/framework/src/ui-framework/shared/ItemDefBase.ts

```diff
@@ -35,7 +35,9 @@ export abstract class ItemDefBase {
   /** @deprecated */
   public isEnabled: boolean = true;
   public isPressed: boolean = false;
-  public isActive: boolean = false;
+  private _isActive: boolean = false;
+  public get isActive(): boolean { return this._isActive; }
+  public set isActive(v: boolean) { this._isActive = v; }

```

Comment:
@DanEastBentley @bsteinbk Is this OK?  Changing this forced me to update a lot of test snapshots...

Reply:
I am OK with the change as it seems harmless. It is sad that only Task and Workflow implement override getter/setter which triggered this change and I don't believe those classes are even used anywhere.